arrocke/gloss-translation
# chore: add vagrant support

## What has changed

- Setup a Vagrantfile to spin up a development environment that has the right node environment, and a postgres database with seed data already set up.
- Documented this in the contributing documentation (with link to new vagrant.md)



## Connected Issues

#83  



## QA Steps
Follow instructions in the `Post-Installation Notes` section of file `vagrant.md`.
1. Make sure you can access database using pgAdmin.
2. Make sure you can ssh to VM.
3. Start API server in VM and make sure you can access via host web browser.
4. Start Web app in VM and make sure you can access via host web browser.


## Post-Deployment




## Known Issue

When accessing Web app via external browser, gets a runtime error when app attempts to get the English gloss for Gen. 1:1.
